lib/email/receiver.rb

```diff
@@ -1406,10 +1424,24 @@ def add_attachments(raw, user, options = {})
           tmp&.close!
         end
       end
+
       if rejected_attachments.present? && !user.staged?
         notify_about_rejected_attachment(rejected_attachments)
       end
 
+      if added_attachments.present?
+        markdown =
+          added_attachments.map { |upload| UploadMarkdown.new(upload).to_markdown }.join("\n")
+        if markdown.present?
+          raw << "\n\n"
+          raw << "[details=\"#{I18n.t("emails.incoming.attachments")}\"]"
+          raw << "\n\n"
+          raw << markdown
+          raw << "\n\n"
+          raw << "[/details]"
+        end
```

Comment:
Pretty minor but maybe it could be nicer to write it like this? (not sure if the double line breaks are needed in this case)

```ruby
raw << <<~DETAILS


[details="#{I18n.t("emails.incoming.attachments")}"]


#{markdown}


[/details]
DETAILS

Reply:
I tried that initially but due to linting the code was ugly... 

Plus, I like that the string version shows the "\n\n" more clearly 🤷‍♂️ 

```ruby
raw << <<~HTML if markdown.present?


  [details="#{I18n.t("emails.incoming.attachments")}"]

  #{markdown}
  
  [/details]
HTML
```

Reply:
Fair enough, yes that looks kind of busted :joy: Was just a thought, your way is totally fine here :+1: